### DISCUSSION TITLE
Why remove save_name parameter?

### DISCUSSION THREAD

**Discussion by kickin6** [mrhan1993/FooocusWithFastAPI#28] [General]
Hi. Great work on the upgraded app. What was the reasoning in removing the save_name parameter?

I modified my fork of Fooocus-API so that save_name was a nested directory in the format:
/output/images/<api-key> where api-key was each users individual files. I had a flask front-end that set save_name to the api-key after validation. This ommission breaks that.

I also don't want to create the directory if it doesn't exist because then anyone could create an apikey directory.

If you're not going to add it, would you consider some other alternative to setting file_path?

Thank you.

**Comment by mrhan1993** [comment]
is back in branch v2.5, after new doc is finish, branch v2.5 will merge to main
